src/routes/(auth)/acceptprojectinvitation/+page.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+    import { goto } from "$app/navigation";
+    import { Button } from "$lib/components/primitives/button";
+    import type { RpcError } from "@protobuf-ts/runtime-rpc";
+    import { GrpcStatusCode } from "@protobuf-ts/grpcweb-transport";
+    import { getGrpcStatusCode } from "$lib/utils/common-helper";
+
+    let { data } = $props();
+    const { acceptancePromise: acceptancePromise } = data;
+
+    type ErrorDetails = { headline: string; body: string } | RpcError;
+
+    function getErrorDetails(error: ErrorDetails): { headline: string; body: string } {
+        const customError = error as { headline: string; body: string };
+        if (customError.headline && customError.headline) {
+            return customError;
+        }
+
+        const rpcError = error as RpcError;
+        const statusCodeValue = getGrpcStatusCode(rpcError.code);
+
+        switch (statusCodeValue) {
+            case GrpcStatusCode.INVALID_ARGUMENT:
+                return {
+                    headline: "Accepting Failed",
+                    body: "This Acceptance link is invalid. Please check the link and try again.",
```

Comment:
```suggestion
                    body: "This acceptance link is invalid. Please check the link and try again.",
```
(same for other bodies)